src/__tests__/__snapshots__/main-test.js.snap

```diff
@@ -267,6 +267,35 @@ Object {
   "displayName": "MyComponent",
   "methods": Array [],
   "props": Object {
+    "OtherComponentProps": Object {
+      "description": "Spread props from another file",
+      "flowType": Object {
+        "name": "OtherComponentProps",
+      },
+      "required": false,
+    },
+    "OtherLocalProps": Object {
```

Comment:
It seems like this could be confused with a prop called `OtherLocalProps` that has these specific keys. However, since it is being spread in, the properties of OtherLocalProps should probably be flattened into the top level props? Is that correct?